File: day00_templates/c/part1.c
#include "helper.h"

#include <stdio.h>
#include <stdlib.h>

int main()
{
    char* *lines = read_lines("input.txt");

    for (int i = 0; i < arrlen(lines); ++i) {
        char* line = lines[i];
        int value = atoi(line);
        // printf("%s\n", line);
        printf("%d\n", value);
    }

    return 0;
}

File: day00_templates/rust/src/bin/part1.rs
// Part 1

use day??::helper::read_input_file;    // where day?? is the project's name from Cargo.toml

const INPUT_FILE: &str = "input.txt";

fn main()
{
    let lines: Vec<String> = helper::read_input_file(INPUT_FILE);
    for line in lines {
        println!("{}", line);
    }
}
